src/pages/video listing/VideoListing.jsx

```diff
@@ -0,0 +1,37 @@
+import classes from './videoListing.module.css'
+import { useEffect, useState } from 'react'
+import axios from 'axios'
+import VideoCard from './components/VideoCard';
+import SideNavbar from '../../components/side navbar/SideNavbar';
+const VideoListing = () => {
+    const [lists,setLists] = useState([]);
+    useEffect(()=>{
+        const fetchVideos=async ()=>{
+            try{
+                const response = await axios.get('/api/videos');
+                setLists(response.data.videos);
```

Comment:
First check `response.status`, if it is 200 then only set your list.

Reply:
sure